prov/rxd/src/rxd_attr.c

```diff
@@ -67,6 +67,7 @@ struct fi_domain_attr rxd_domain_attr = {
 	.data_progress = FI_PROGRESS_MANUAL,
 	.resource_mgmt = FI_RM_ENABLED,
 	.av_type = FI_AV_UNSPEC,
+	.mr_mode = FI_MR_LOCAL | OFI_MR_BASIC_MAP,
```

Comment:
This should really be FI_MR_BASIC | FI_MR_SCALABLE.  See the rxm_attr.c code for how it handles this.  RxD itself has no requirements.  Those come from the underlying provider.  This entire area likely needs work.

Reply:
With `FI_MR_BASIC | FI_MR_SCALABLE`, does the layering logic know enough to pass through the modes from the core provider sitting below and match applications appropriately (for instance, when the core provider expects FI_MR_LOCAL and RxD does not really expose that)?

Reply:
From the perspective of RxD, it doesn't require any modes.  All modes are basically derived from the requirements of the underlying provider.  That's where things get messy.  RxD needs to pass through the core provider requirements to the app without adding any.  In some cases, it may be able to remove a core provider requirement and not pass that along to the app.  RxM acts in a similar fashion.  I reference rxm because we've spent a lot more time getting its layering correct based on different core provider requirements.  It has more complete handling for version differences as well.